pkg/didcomm/protocol/didexchange/service.go

```diff
@@ -65,7 +65,8 @@ type provider interface {
 }
 
 type connectionStore interface {
-	GetConnection(connectionID string) (*ConnectionRecord, error)
+	GetConnectionRecord(connectionID string) (*ConnectionRecord, error)
+	SaveConnectionRecord(record *ConnectionRecord) error
```

Comment:
Does `SaveConnectionRecord()` really need to be exported?

Reply:
yea i dont think it needs to be so i have updated it be a private function.